diff --git a/base/BUILD.gn b/base/BUILD.gn
index e269f3a0bee06dec36300af0ea2a2181f597011a..a8cd52ca8024f02b7fc46e9ee9837034495d43a9 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5248,6 +5248,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5974,6 +5975,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
