diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index a55e5c5ae64dabc3353f74407b9dea133d018a19..8820459cc330bf7514c3d5222f54072befd36f18 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -412,6 +412,11 @@ if (!is_robolectric && enable_java_templates) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) {
+      invoker.includes += brave_android_manifest_includes
+    } else {
+      invoker.includes = brave_android_manifest_includes
+    }
     action_with_pydeps(target_name) {
       forward_variables_from(invoker, TESTONLY_AND_VISIBILITY + [ "deps" ])
       inputs = [ invoker.input ]
