diff --git a/chrome/browser/extensions/extension_management.cc b/chrome/browser/extensions/extension_management.cc
index 9221fb734c01c1a47ca96845a7810a899376a49a..acdd5fde83a5935298ad6e5b667ed35054e50842 100644
--- a/chrome/browser/extensions/extension_management.cc
+++ b/chrome/browser/extensions/extension_management.cc
@@ -1180,7 +1180,7 @@ ExtensionManagementFactory::BuildServiceInstanceForBrowserContext(
     content::BrowserContext* context) const {
   TRACE_EVENT0("browser,startup",
                "ExtensionManagementFactory::BuildServiceInstanceFor");
-  return std::make_unique<ExtensionManagement>(
+  return std::make_unique<BraveExtensionManagement>(
       Profile::FromBrowserContext(context));
 }
 
