diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index a4ae83016aa4b02f39e3905eb51c9e95e3f9a438..9d6727828adea8e051875102fd6a32dd83f4ef98 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1404,3 +1404,5 @@ if (enable_screen_capture) {
     "java/src/org/chromium/chrome/browser/media/TabSharingUiManager.java",
   ]
 }
+chrome_java_sources -=
+    [ "java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java" ]
