diff --git a/chrome/browser/ui/actions/chrome_action_id.h b/chrome/browser/ui/actions/chrome_action_id.h
index 6a0d7d37eb1f4e7a4f93924aa37d49b9e6bc5210..d7740b499440fc14543bd6d06585f0c2102aacae 100644
--- a/chrome/browser/ui/actions/chrome_action_id.h
+++ b/chrome/browser/ui/actions/chrome_action_id.h
@@ -497,6 +497,10 @@
 // Should be reinserted into CHROME_COMMON_ACTION_IDS when general solution to
 // add action id mappings is implemented.
 #define SIDE_PANEL_ACTION_IDS \
+  E(kActionSidePanelShowPlaylist) \
+  E(kActionSidePanelShowChatUI) \
+  E(kActionSidePanelShowBraveNews) \
+  E(kActionSidePanelShowWallet) \
   /* Side Panel items */ \
   E(kActionSidePanelShowAboutThisSite) \
   E(kActionSidePanelShowAssistant) \
@@ -524,6 +528,13 @@
   E(kActionSidePanelShowMerchantTrust)
 
 #define TOOLBAR_PINNABLE_ACTION_IDS \
+  E(kActionShowPartitionedStorage) \
+  E(kActionShowPsstIcon) \
+  E(kActionShowSpeedreader) \
+  E(kActionShowWaybackMachine) \
+  E(kActionShowPlaylistPageAction) \
+  E(kActionShowOnionLocation) \
+  E(kActionShowBraveNews) \
   E(kActionHome, IDC_HOME) \
   E(kActionForward, IDC_FORWARD) \
   E(kActionNewIncognitoWindow, IDC_NEW_INCOGNITO_WINDOW) \
