diff --git a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
index 06214c17edd45655633e3cf13ad18141cf78db1a..aa421d747f5f910c96cf9e30c3313d0be4a45cfe 100644
--- a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
+++ b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabSwitcherPaneCoordinator.java
@@ -587,7 +587,7 @@ public class TabSwitcherPaneCoordinator implements BackPressHandler {
                     recyclerViewTimer.getElapsedMillis());
 
             mTabListEditorManager =
-                    new TabListEditorManager(
+                    new BraveTabListEditorManager(
                             activity,
                             mModalDialogManager,
                             coordinatorView,
