diff --git a/chrome/browser/extensions/api/identity/identity_get_auth_token_function.h b/chrome/browser/extensions/api/identity/identity_get_auth_token_function.h
index 523d893998efa31a1c72db2db53a1183cf03b2e6..dbdb5c3ff9da6b1aebdbe77548bee48cf404612c 100644
--- a/chrome/browser/extensions/api/identity/identity_get_auth_token_function.h
+++ b/chrome/browser/extensions/api/identity/identity_get_auth_token_function.h
@@ -161,7 +161,7 @@ class IdentityGetAuthTokenFunction : public ExtensionFunction,
   FRIEND_TEST_ALL_PREFIXES(GetAuthTokenFunctionTest, NoninteractiveShutdown);
 
   class RefreshTokensLoadedWaiter;
-#if BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#if BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
   class AccountsInCookieUpdatedWaiter;
 #endif
   enum class InteractionType { kSignin, kConsent };
@@ -174,7 +174,7 @@ class IdentityGetAuthTokenFunction : public ExtensionFunction,
   // might be started.
   void GetAuthTokenForAccount(const GaiaId& gaia_id);
 
-#if BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#if BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
   void OnCookiesUpdatedForRemoteConsent(bool success);
   bool ShouldDelayRemoteConsent();
   void StartWaitingForCookies();
@@ -293,7 +293,7 @@ class IdentityGetAuthTokenFunction : public ExtensionFunction,
   // a permissions prompt will be popped up to the user.
   RemoteConsentResolutionData resolution_data_;
   std::unique_ptr<RefreshTokensLoadedWaiter> refresh_tokens_loaded_waiter_;
-#if BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#if BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
   std::unique_ptr<AccountsInCookieUpdatedWaiter>
       accounts_in_cookie_updated_waiter_;
 #endif
