diff --git a/chrome/browser/net/system_network_context_manager.cc b/chrome/browser/net/system_network_context_manager.cc
index 5137f4386d9572940c5dbf2e554e4cfddc4f7724..e9df77ef998e11f29c64f91a68887cde5f291701 100644
--- a/chrome/browser/net/system_network_context_manager.cc
+++ b/chrome/browser/net/system_network_context_manager.cc
@@ -488,7 +488,7 @@ class SystemNetworkContextManager::URLLoaderFactoryForSystem
       return;
     }
     manager_->GetURLLoaderFactory()->CreateLoaderAndStart(
-        std::move(receiver), request_id, options, url_request,
+        std::move(receiver), request_id, options, BRAVE_CREATE_LOADER_AND_START,
         std::move(client), traffic_annotation);
   }
 
