diff --git a/chrome/browser/ui/android/bottombar/java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarCoordinator.java b/chrome/browser/ui/android/bottombar/java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarCoordinator.java
index 17681f2956cebd63fe9d6fa5cdf3532ec4b39b0d..52d1f0c3df540b24e09af258421d8f246e2a35ff 100644
--- a/chrome/browser/ui/android/bottombar/java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarCoordinator.java
+++ b/chrome/browser/ui/android/bottombar/java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarCoordinator.java
@@ -43,7 +43,7 @@ public class BottomBarCoordinator implements BottomBar, Destroyable {
     private final BottomBarMediator mMediator;
     private final BottomBarView mView;
     private final PropertyModelChangeProcessor<PropertyModel, BottomBarView, PropertyKey> mMcp;
-    private final BottomBarButtonManager mButtonManager;
+    protected final BottomBarButtonManager mButtonManager;
     private final BottomBarPromoDialogCoordinator mPromoDialogCoordinator;
     private final NullableObservableSupplier<Tab> mTabSupplier;
 
@@ -89,7 +89,7 @@ public class BottomBarCoordinator implements BottomBar, Destroyable {
         mModel = new PropertyModel.Builder(BottomBarProperties.ALL_KEYS).build();
 
         mButtonManager =
-                new BottomBarButtonManager(configs, actionRegistry, mModel, ActionId.NEW_TAB);
+                new BraveBottomBarButtonManager(configs, actionRegistry, mModel, ActionId.NEW_TAB);
 
         mPromoDialogCoordinator =
                 new BottomBarPromoDialogCoordinator(
@@ -117,7 +117,7 @@ public class BottomBarCoordinator implements BottomBar, Destroyable {
         mMcp = PropertyModelChangeProcessor.create(mModel, mView, BottomBarViewBinder::bind);
     }
 
-    private List<ActionConfig> createActionConfigs(
+    protected List<ActionConfig> createActionConfigs(
             BottomBarView view, boolean shouldIncludeHomeButton) {
         List<ActionConfig> configs = new ArrayList<>();
 
