diff --git a/chrome/browser/ui/views/autofill/popup/popup_view_views.cc b/chrome/browser/ui/views/autofill/popup/popup_view_views.cc
index e436528136a2f2c674570cc03a844963ebf842f0..45793059c875419156457ec4b9b66aa9d73f33a1 100644
--- a/chrome/browser/ui/views/autofill/popup/popup_view_views.cc
+++ b/chrome/browser/ui/views/autofill/popup/popup_view_views.cc
@@ -1349,7 +1349,7 @@ void PopupViewViews::InitViews() {
         ImageFetcherServiceFactory::GetForKey(profile->GetProfileKey())
             ->GetImageFetcher(
                 image_fetcher::ImageFetcherConfig::kInMemoryWithDiskCache);
-    password_favicon_loader_ = std::make_unique<PasswordFaviconLoaderImpl>(
+    password_favicon_loader_ = CreatePasswordFaviconLoader(profile,
         favicon_service, image_fetcher);
   }
 
