diff --git a/chrome/common/webui_url_constants.cc b/chrome/common/webui_url_constants.cc
index e4ebcfc6c13dc01fc01c35e264fb3fc96c8f7bb9..a8c6d63455aa4cd4c3a99b42bb8f4f629a25d732 100644
--- a/chrome/common/webui_url_constants.cc
+++ b/chrome/common/webui_url_constants.cc
@@ -289,6 +289,18 @@ base::span<const base::cstring_view> ChromeURLHosts() {
       kChromeUIOrganizerPanelHost,
       kChromeUIWebuiBrowserHost,
 #endif  // !BUILDFLAG(IS_ANDROID)
+      kAdblockHost,
+      kAdblockInternalsHost,
+      kRewardsPageHost,
+      kRewardsInternalsHost,
+      kWelcomeHost,
+#if BUILDFLAG(ENABLE_BRAVE_WALLET)
+      kWalletPageHost,
+#endif
+      kTorInternalsHost,
+      kSkusInternalsHost,
+      kAdsInternalsHost,
+      kNewTabTakeoverHost,
   });
 
   return base::span(kChromeURLHosts);
