diff --git a/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc b/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc
index a092efcce09a46bb1400473139ef30dfe65acb67..51d9d0a170ea9ccde089816864c760b9f17f47d2 100644
--- a/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc
+++ b/chrome/browser/profiles/chrome_browser_main_extra_parts_profiles.cc
@@ -700,7 +700,7 @@ void ChromeBrowserMainExtraPartsProfiles::
   chromeos_extensions::EnsureBrowserContextKeyedServiceFactoriesBuilt();
 #endif
   extensions::EnsureBrowserContextKeyedServiceFactoriesBuilt();
-#elif BUILDFLAG(ENABLE_DESKTOP_ANDROID_EXTENSIONS)
+#elif BUILDFLAG(IS_ANDROID) && BUILDFLAG(ENABLE_EXTENSIONS_CORE)
   chrome_extensions::EnsureBrowserContextKeyedServiceFactoriesBuilt();
   // EnsureBrowserContextKeyedServiceFactoriesBuilt() is invoked before the
   // ExtensionsBrowserClient is ready on Android. This is due to Android
