diff --git a/chrome/browser/ui/autofill/autofill_context_menu_manager.cc b/chrome/browser/ui/autofill/autofill_context_menu_manager.cc
index c816b56d1e09a2b9fea54b323b5b08c991f39b40..2fa4f8f242006e25d5538b2ee18887bd578395ef 100644
--- a/chrome/browser/ui/autofill/autofill_context_menu_manager.cc
+++ b/chrome/browser/ui/autofill/autofill_context_menu_manager.cc
@@ -147,6 +147,7 @@ AutofillContextMenuManager::AutofillContextMenuManager(
 AutofillContextMenuManager::~AutofillContextMenuManager() = default;
 
 void AutofillContextMenuManager::AppendItems() {
+  [&]() -> void {
   if (params_.is_content_editable_for_autofill) {
     if (MaybeAddAtMemoryItem()) {
       menu_model_->AddSeparator(ui::NORMAL_SEPARATOR);
@@ -156,6 +157,8 @@ void AutofillContextMenuManager::AppendItems() {
 
   MaybeAddAutofillManualFallbackItems();
   MaybeAddAutofillFeedbackItem();
+  }();
+  RemoveAutofillFeedbackMenuItem(menu_model_);
 }
 
 bool AutofillContextMenuManager::IsCommandIdSupported(int command_id) {
