diff --git a/chrome/browser/extensions/chrome_extensions_browser_client.cc b/chrome/browser/extensions/chrome_extensions_browser_client.cc
index d5b5ab4e8c3aa742a443caf75c5a2b1ffa8d9bea..cacca2a05c10ecad97f9766e74ead2d58c1a97d9 100644
--- a/chrome/browser/extensions/chrome_extensions_browser_client.cc
+++ b/chrome/browser/extensions/chrome_extensions_browser_client.cc
@@ -276,7 +276,7 @@ bool RegisterTransformers() {
 ChromeExtensionsBrowserClient::ChromeExtensionsBrowserClient()
     : resource_manager_(
           std::make_unique<ChromeComponentExtensionResourceManager>()),
-      api_client_(std::make_unique<ChromeExtensionsAPIClient>()),
+      api_client_(std::make_unique<BraveExtensionsAPIClient>()),
       event_router_forwarder_(base::MakeRefCounted<EventRouterForwarder>()) {
   AddAPIProvider(std::make_unique<CoreExtensionsBrowserAPIProvider>());
   AddAPIProvider(std::make_unique<ChromeExtensionsBrowserAPIProvider>());
@@ -676,6 +676,10 @@ void ChromeExtensionsBrowserClient::AttachExtensionTaskManagerTag(
 scoped_refptr<update_client::UpdateClient>
 ChromeExtensionsBrowserClient::CreateUpdateClient(
     scoped_refptr<update_client::Configurator> configurator) {
+  if (base::FeatureList::IsEnabled(
+    extensions_features::kBraveSequentialExtensionsUpdate)) {
+    return update_client::SequentialUpdateClientFactory(configurator);
+  }
   return update_client::UpdateClientFactory(configurator);
 }
 
@@ -691,7 +695,7 @@ ChromeExtensionsBrowserClient::CreateUpdateClientConfigurator(
       override_url = update_url;
     }
   }
-  return ChromeUpdateClientConfig::Create(context, override_url);
+  return BraveUpdateClientConfig::Create(context, override_url);
 }
 
 std::unique_ptr<ScopedBrowserContextKeepAlive>
