diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
index 13569ea6785f8cffebe28a2f9e77c55b9b156d90..dbc952ae6867ce92a82a1dc64eaa60679060f85d 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
@@ -283,8 +283,8 @@ public class ToolbarPhone extends ToolbarLayout
     private @Nullable PhoneCaptureStateToken mPhoneCaptureStateToken;
     private @Nullable ButtonData mButtonData;
 
-    private final @ColorInt int mToolbarBackgroundColorForNtp;
-    private final @ColorInt int mLocationBarBackgroundColorForNtp;
+    @ColorInt int mToolbarBackgroundColorForNtp;
+    @ColorInt int mLocationBarBackgroundColorForNtp;
     private boolean mUseAdjustedTintColorForNtp;
     private boolean mIsToolbarExpandedOnNtp;
 
@@ -576,6 +576,7 @@ public class ToolbarPhone extends ToolbarLayout
 
     /** Set the background color of the location bar to appropriately match the theme color. */
     private void updateModernLocationBarColor(@ColorInt int color) {
+        BraveToolbarLayout.class.cast(this).updateModernLocationBarColorImpl(color);
         if (mCurrentLocationBarColor == color) return;
         mCurrentLocationBarColor = color;
         mLocationBarBackground.setBackgroundColor(color);
@@ -1291,6 +1292,7 @@ public class ToolbarPhone extends ToolbarLayout
      * Tab Page.
      */
     private void updateLocationBarLayoutForExpansionAnimation() {
+        if (!BraveToolbarLayout.class.cast(this).isLocationBarValid(mLocationBar)) return;
         TraceEvent.begin("ToolbarPhone.updateLocationBarLayoutForExpansionAnimation");
         if (isInTabSwitcherMode()) return;
 
@@ -1707,6 +1709,7 @@ public class ToolbarPhone extends ToolbarLayout
             }
         }
 
+        BraveToolbarLayout.class.cast(this).drawAnimationOverlay(mToolbarButtonsContainer, canvas);
         // Draw the tab stack button and associated text if necessary.
         ToggleTabStackButtonCoordinator tabSwitcherButtonCoordinator =
                 getTabSwitcherButtonCoordinator();
@@ -2099,7 +2102,7 @@ public class ToolbarPhone extends ToolbarLayout
 
     @Override
     public void onHomeButtonIsEnabledUpdate(boolean homeButtonEnabled) {
-        mIsHomeButtonEnabled = homeButtonEnabled;
+        mIsHomeButtonEnabled = false;
         updateButtonVisibility();
     }
 
@@ -2505,6 +2508,7 @@ public class ToolbarPhone extends ToolbarLayout
     }
 
     private void createAndRunFocusAnimatorLegacy(boolean hasFocus, boolean shouldShowKeyboard) {
+        if (!BraveToolbarLayout.class.cast(this).isLocationBarValid(mLocationBar)) return;
         List<Animator> animators = new ArrayList<>();
         if (hasFocus) {
             populateUrlExpansionAnimatorSet(animators);
@@ -3668,6 +3672,7 @@ public class ToolbarPhone extends ToolbarLayout
      * @return The offset for the location bar when showing the dse icon.
      */
     int getLocationBarOffsetForFocusAnimation(boolean hasFocus) {
+        if (!urlHasFocus()) return 0;
         StatusCoordinator statusCoordinator = mLocationBar.getStatusCoordinator();
         if (statusCoordinator == null) return 0;
 
