diff --git a/chrome/browser/autofill/android/BUILD.gn b/chrome/browser/autofill/android/BUILD.gn
index 431e7a4daec9ea731190d98779a1068ba685fbb6..706aee6bc6603eb111ef0c7e2208a73864c2b86a 100644
--- a/chrome/browser/autofill/android/BUILD.gn
+++ b/chrome/browser/autofill/android/BUILD.gn
@@ -125,6 +125,7 @@ android_library("java") {
   deps = [
     ":java_resources",
     "//base:base_java",
+    "//brave/android:settings_insets_java",
     "//chrome/android:chrome_app_java_resources",
     "//chrome/browser/device_reauth/android:java",
     "//chrome/browser/feedback/android:factory_java",
