diff --git a/chrome/browser/ui/android/bottombar/java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarConfigUtils.java b/chrome/browser/ui/android/bottombar/java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarConfigUtils.java
index dd9cd479ebe65e80d4e331eaefd1eb2170672afb..263a03d53a6dcad6f96a1fc3b4dc3f93eb3a87bd 100644
--- a/chrome/browser/ui/android/bottombar/java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarConfigUtils.java
+++ b/chrome/browser/ui/android/bottombar/java/src/org/chromium/chrome/browser/ui/bottombar/BottomBarConfigUtils.java
@@ -32,6 +32,7 @@ public class BottomBarConfigUtils {
     // LINT.IfChange(isBottomBarEnabled)
     /** Whether the bottom bar is enabled. */
     public static boolean isBottomBarEnabled(Context context) {
+        if (!BraveBottomBarUserPrefs.isBottomBarEnabled()) return false;
         return !DeviceFormFactor.isNonMultiDisplayContextOnTablet(context)
                 && !DeviceInfo.isAutomotive()
                 && ChromeFeatureList.sAndroidBottomBar.isEnabled();
