diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
index 7b861520e52fa728346a4ee1e65cce0a15a62dee..d4ae555b6ca5be80c9a71c7ffcb04d8bcfe1c90f 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
@@ -318,6 +318,7 @@ public class ToolbarTablet extends ToolbarLayout {
         mFixedHeightBackground.setBackgroundColor(color);
         mLocationBar.updateVisualsForState();
         setToolbarHairlineColor(color);
+        super.onThemeColorChanged(color, shouldAnimate);
 
         // Notify the StatusBarColorController of the toolbar color change. This is to match the
         // status bar's color with the toolbar color when the tab strip is hidden on a tablet.
