diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index 9ba24f07dcee786cc1c187453a69b62b37b7d95e..d73c9e2f33cd6cc0b43d692e9535f423866a0884 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -990,6 +990,14 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
 
                         private void closeIfNoTabsAndHomepageEnabled(
                                 boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
+                            if (getTabModelSelector().getTotalTabCount() == 0) {
+                                shouldRemoveWindowWithZeroTabs = shouldRemoveWindowWithZeroTabs && org.chromium.chrome.browser.partnercustomizations.CloseBraveManager.shouldCloseAppWithZeroTabs();
+                            }
+                            closeIfNoTabsAndHomepageEnabled_ChromiumImpl(isPendingClosure, shouldRemoveWindowWithZeroTabs);
+                        }
+
+                        private void closeIfNoTabsAndHomepageEnabled_ChromiumImpl(
+                                boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
                             if (getTabModelSelector().getTotalTabCount() == 0) {
                                 if (shouldRemoveWindowWithZeroTabs) {
                                     mMultiInstanceManager.closeChromeWindowIfEmpty(mWindowId);
@@ -3981,6 +3989,11 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
                 new TabModelSelectorObserver() {
                     @Override
                     public void onTabStateInitialized() {
+                        onTabStateInitialized_ChromiumImpl();
+                        onTabStateInitializedHandler();
+                    }
+
+                    private void onTabStateInitialized_ChromiumImpl() {
                         if (mMultiInstanceManager != null) {
                             mMultiInstanceManager.onTabStateInitialized();
                         }
