diff --git a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java
index 660da722c16d66e66245550b442a56160e6cef59..3306823df3fdd62c1f14a066a8692fc2f1857856 100644
--- a/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java
+++ b/chrome/android/features/tab_ui/java/src/org/chromium/chrome/browser/tasks/tab_management/TabGridDialogMediator.java
@@ -597,7 +597,7 @@ public class TabGridDialogMediator
                                 Set.of(TabListEditorItemSelectionId.createTabId(tabId)));
                     };
             mTabGridContextMenuCoordinator =
-                    new TabGridContextMenuCoordinator(
+                    new BraveTabGridContextMenuCoordinator(
                             activity,
                             tabBookmarkerSupplier,
                             profile,
