diff --git a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_browsertest.cc b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_browsertest.cc
index f4cc0e361431dd26296065fdb677b629a52ae576..4851dcd1bc24068f3866328d709ea6c8a8bafe59 100644
--- a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_browsertest.cc
+++ b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_browsertest.cc
@@ -203,6 +203,9 @@ class RecentTabsSubMenuModelTest : public InProcessBrowserTest {
     return new_browser;
   }
 
+  void VerifyModel(const RecentTabsSubMenuModel& model, base::span<const ModelData> data);
+  void VerifyModel(const ui::MenuModel* model, base::span<const ModelData> data);
+
  private:
   std::unique_ptr<syncer::DataTypeProcessor> sync_processor_;
 };
@@ -242,7 +245,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 }
 
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       LogMenuMetricsForShowGroupedHistory) {
+                       DISABLED_LogMenuMetricsForShowGroupedHistory) {
   Init();
   FakeIconDelegate fake_delegate;
   AppMenuIconController app_menu_icon_controller(browser()->GetProfile(),
@@ -334,7 +337,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest, NoTabs) {
 
 // Test enabled "Recently closed" header with no foreign tabs.
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       RecentlyClosedTabsFromCurrentSession) {
+                       DISABLED_RecentlyClosedTabsFromCurrentSession) {
   Init();
   DisableSync();
 
@@ -373,7 +376,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 
 // Test recently closed groups with no foreign tabs.
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       RecentlyClosedGroupsFromCurrentSession) {
+                       DISABLED_RecentlyClosedGroupsFromCurrentSession) {
   Init();
   ASSERT_TRUE(browser()->tab_strip_model()->SupportsTabGroups());
 
@@ -430,7 +433,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 }
 
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       RecentlyClosedSplitsFromCurrentSession) {
+                       DISABLED_RecentlyClosedSplitsFromCurrentSession) {
   Init();
   ASSERT_TRUE(browser()->tab_strip_model()->SupportsTabGroups());
 
@@ -480,7 +483,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 }
 
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       RecentlyClosedWindowWithSplit) {
+                       DISABLED_RecentlyClosedWindowWithSplit) {
   Init();
   DisableSync();
 
@@ -531,7 +534,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 }
 
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       RecentlyClosedWindowWithGroupsAndSplits) {
+                       DISABLED_RecentlyClosedWindowWithGroupsAndSplits) {
   Init();
   DisableSync();
 
@@ -633,7 +636,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 }
 
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       RecentlyClosedWindowWithSplitAndRegularTabs) {
+                       DISABLED_RecentlyClosedWindowWithSplitAndRegularTabs) {
   Init();
   DisableSync();
 
@@ -690,7 +693,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 }
 
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       RecentlyClosedGroupWithSplit) {
+                       DISABLED_RecentlyClosedGroupWithSplit) {
   Init();
   ASSERT_TRUE(browser()->tab_strip_model()->SupportsTabGroups());
 
@@ -748,7 +751,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 }
 
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       RecentlyClosedTabsAndWindowsFromLastSessionWithRefresh) {
+                       DISABLED_RecentlyClosedTabsAndWindowsFromLastSessionWithRefresh) {
   Init();
   DisableSync();
 
@@ -936,7 +939,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest, OtherDevicesDynamicUpdate) {
   VerifyModel(model, kDataAfterUpdate);
 }
 
-IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest, MaxSessionsAndRecency) {
+IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest, DISABLED_MaxSessionsAndRecency) {
   Init();
   EnableSync();
 
@@ -975,7 +978,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest, MaxSessionsAndRecency) {
 }
 
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       MaxTabsPerSessionAndRecency) {
+                       DISABLED_MaxTabsPerSessionAndRecency) {
   Init();
   EnableSync();
 
@@ -1019,7 +1022,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 }
 
 IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
-                       SavingBrowserHistoryDisabledPolicyChangeMidSession) {
+                       DISABLED_SavingBrowserHistoryDisabledPolicyChangeMidSession) {
   Init();
   DisableSync();
   // Ensure the policy is explicitly false to start with.
@@ -1102,7 +1105,7 @@ IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest,
 }
 
 #if !BUILDFLAG(IS_CHROMEOS)
-IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest, OtherDevicesAvailability) {
+IN_PROC_BROWSER_TEST_F(RecentTabsSubMenuModelTest, DISABLED_OtherDevicesAvailability) {
   if (!syncer::IsReplaceSyncPromosWithSignInPromosEnabled()) {
     GTEST_SKIP();
   }
