diff --git a/chrome/browser/ui/views/profiles/BUILD.gn b/chrome/browser/ui/views/profiles/BUILD.gn
index 56dbd77d05f2d8404fa181a1dc75085ebdeb76fe..83a9243ae2e1484db61ffd4456bfa5c1a309ab69 100644
--- a/chrome/browser/ui/views/profiles/BUILD.gn
+++ b/chrome/browser/ui/views/profiles/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/ui/webui/custom_profile_image/buildflags/buildflags.gni")
 import("//build/config/ui.gni")
 import("//components/safe_browsing/buildflags.gni")
 import("//components/signin/features.gni")
@@ -231,6 +232,13 @@ if (is_win || is_mac || is_linux || is_chromeos) {
     if (enable_dice_support) {
       deps += [ "//chrome/browser/ui/webui/signin/signout_confirmation" ]
     }
+    deps += [ "//brave/ui/webui/custom_profile_image/buildflags" ]
+    if (enable_brave_custom_profile_image_webui) {
+      deps += [
+        "//brave/browser/ui/webui/custom_profile_image",
+        "//chrome/browser:file_select_helper",
+      ]
+    }
   }
 }
 
