diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index 957cbc93e25ad807ce35360a2a7e1240e8f5ce92..3faddbe091d8f264ae0d2320bfe78df5177ad40a 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -1334,6 +1334,7 @@ void AutocompleteController::InitializeAsyncProviders(int provider_types) {
         base::MakeRefCounted<SearchProvider>(provider_client_.get(), this);
     search_provider_ = search_provider.get();
     providers_.push_back(std::move(search_provider));
+    BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // Providers run in the order they're added.  Add `HistoryURLProvider` after
   // `SearchProvider` because:
@@ -1632,6 +1633,7 @@ void AutocompleteController::UpdateResult(UpdateType update_type,
     StartStopTimer();
   }
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
   PostProcessMatches();
   // TODO (crbug.com/555355466): Refactor or remove this logic when we remove
   // OB simplification code paths.
