diff --git a/components/autofill/core/common/autofill_features.h b/components/autofill/core/common/autofill_features.h
index beab812752e00c7c4fe1cbcc014dc923631bd031..79fa8e7483de659a5d5cd1a8098545236d998a63 100644
--- a/components/autofill/core/common/autofill_features.h
+++ b/components/autofill/core/common/autofill_features.h
@@ -110,7 +110,7 @@ COMPONENT_EXPORT(AUTOFILL)
 BASE_DECLARE_FEATURE_PARAM(std::string,
                            kAutofillAiSavePromptSurveyDeclinedTriggerId);
 COMPONENT_EXPORT(AUTOFILL)
-DECLARE_FEATURE_WITH_MOBILE_COUNTRY_RESTRICTION(kAutofillAiServerModel);
+BASE_DECLARE_FEATURE(kAutofillAiServerModel);
 COMPONENT_EXPORT(AUTOFILL)
 BASE_DECLARE_FEATURE_PARAM(base::TimeDelta, kAutofillAiServerModelCacheAge);
 COMPONENT_EXPORT(AUTOFILL)
@@ -164,7 +164,7 @@ COMPONENT_EXPORT(AUTOFILL)
 DECLARE_FEATURE_WITH_MOBILE_COUNTRY_RESTRICTION(
     kAutofillAiWalletVehicleRegistration);
 COMPONENT_EXPORT(AUTOFILL)
-DECLARE_FEATURE_WITH_MOBILE_COUNTRY_RESTRICTION(kAutofillAiWithDataSchema);
+BASE_DECLARE_FEATURE(kAutofillAiWithDataSchema);
 COMPONENT_EXPORT(AUTOFILL)
 BASE_DECLARE_FEATURE(kAutofillAllowFillingModifiedInitialValues);
 COMPONENT_EXPORT(AUTOFILL)
