diff --git a/components/password_manager/core/browser/features/password_features.cc b/components/password_manager/core/browser/features/password_features.cc
index e9ac3f19027a90607988a8dff7c9d88799cee9ae..52c437523b2b481a5ae07bd3f2203dfa006c7386 100644
--- a/components/password_manager/core/browser/features/password_features.cc
+++ b/components/password_manager/core/browser/features/password_features.cc
@@ -210,11 +210,11 @@ const base::FeatureParam<int> kCapturePageContentRetryCount = {
 BASE_FEATURE(kShowTabWithPasswordChangeOnSuccess,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
-BASE_FEATURE(kSkipUndecryptablePasswords,
-#if BUILDFLAG(IS_WIN)
-             base::FEATURE_ENABLED_BY_DEFAULT
+BASE_OVERRIDDEN_FEATURE(kSkipUndecryptablePasswords,
+#if BUILDFLAG(IS_IOS) || BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_WIN)
+  base::FEATURE_ENABLED_BY_DEFAULT
 #else
-             base::FEATURE_DISABLED_BY_DEFAULT
+  base::FEATURE_DISABLED_BY_DEFAULT
 #endif
 );
 
