diff --git a/content/browser/preloading/prefetch/prefetch_features.cc b/content/browser/preloading/prefetch/prefetch_features.cc
index ded261c7694467736b6a1ed0f1a2567482c6ed2a..f9cf0d2b0ef0903359b0c03621db85d1754f61d3 100644
--- a/content/browser/preloading/prefetch/prefetch_features.cc
+++ b/content/browser/preloading/prefetch/prefetch_features.cc
@@ -19,7 +19,7 @@ const base::FeatureParam<int> kPrefetchReusableBodySizeLimit{
 
 BASE_FEATURE(kPrefetchNIKScope, base::FEATURE_DISABLED_BY_DEFAULT);
 
-BASE_FEATURE(kPrefetchClientHints, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kPrefetchClientHints, base::FEATURE_DISABLED_BY_DEFAULT);
 
 constexpr base::FeatureParam<PrefetchClientHintsCrossSiteBehavior>::Option
     kPrefetchClientHintsCrossSiteBehaviorOptions[] = {
@@ -46,7 +46,7 @@ BASE_FEATURE(kPrefetchNetworkPriorityForEmbedders,
 BASE_FEATURE(kPrefetchBumpNetworkPriorityAfterBeingServed,
              base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kPrefetchServiceWorker, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kPrefetchServiceWorker, base::FEATURE_DISABLED_BY_DEFAULT);
 
 bool IsPrefetchServiceWorkerEnabled(content::BrowserContext* browser_context) {
   return base::FeatureList::IsEnabled(kPrefetchServiceWorker) &&
