diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index a4e335cacb1ac1064968a0b2a21e4ebd780c3eae..e07d65ff3811d725d4dde4226e339476addb90e9 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/content/browser/sources.gni")
 import("//base/allocator/partition_allocator/partition_alloc.gni")
 import("//build/buildflag_header.gni")
 import("//build/config/compiler/pgo/pgo.gni")
@@ -383,6 +384,7 @@ source_set("browser") {
     "//v8",
     "//v8:v8_version",
   ]
+  deps += brave_content_browser_deps
 
   public_deps = [
     "//base",
@@ -2381,6 +2383,7 @@ source_set("browser") {
     "worker_host/worker_util.cc",
     "worker_host/worker_util.h",
   ]
+  sources += brave_content_browser_sources
 
   if (is_android) {
     sources += [
@@ -3653,6 +3656,7 @@ group("for_content_tests") {
     "//content/test/*",
     "//content/web_test:web_test_browser",
   ]
+  visibility += brave_content_browser_for_content_tests_visibility
   if (!is_component_build) {
     public_deps = [ ":browser" ]
   } else {
