diff --git a/DEPS b/DEPS
index b7f51fbce1882943e186900b77d3cd9d2d2a30b4..4e423ccce462d0ab90c58d44d5f05b41b41b2d08 100644
--- a/DEPS
+++ b/DEPS
@@ -4327,7 +4327,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
