diff --git a/extensions/common/extension_features.cc b/extensions/common/extension_features.cc
index b7c950fe85659dbba5eb8d7048f0d858de14e8c0..f312bcc336aaa7d3d8faeb74feebd830687dc5fe 100644
--- a/extensions/common/extension_features.cc
+++ b/extensions/common/extension_features.cc
@@ -31,7 +31,7 @@ BASE_FEATURE(kApiOdfsConfigPrivate, base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kApiContextualTasksPrivate, base::FEATURE_DISABLED_BY_DEFAULT);
 
-BASE_FEATURE(kApiGlicPrivate, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kApiGlicPrivate, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kApiEnterpriseReportingPrivateOnDataMaskingRulesTriggered,
              base::FEATURE_ENABLED_BY_DEFAULT);
@@ -92,8 +92,8 @@ BASE_FEATURE(kWebRequestSecurityInfo, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kWebRequestPerContextEventDispatch,
              base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kApiDesktopAndroidNativeMessaging,
-             base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_OVERRIDDEN_FEATURE(kApiDesktopAndroidNativeMessaging,
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kApiDesktopAndroidNativeMessagingBypassExtensionAllowlist,
              base::FEATURE_DISABLED_BY_DEFAULT);
