diff --git a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
index f0067f703b82e43a3ee6f2a833edc0590b231328..921c5f57e1a61bc58d56d243739adde79e4121ae 100644
--- a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
@@ -485,6 +485,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   // Deprecated 08/2026.
   registry->RegisterTimePref(kWaitingForMultiProfileForcedMigrationTimestamp,
                              base::Time());
+  brave::RegisterLocalStatePrefs(registry);
 }
 
 void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -929,6 +930,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   // Deprecated 09/2026.
   registry->RegisterDictionaryPref(kSigninHasAcceptedManagementDialog);
   registry->RegisterBooleanPref(kFingerprintingProtectionEnabled, false);
+  brave::RegisterBrowserStatePrefs(registry);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -960,6 +962,7 @@ void MigrateObsoleteLocalStatePrefs(PrefService* prefs) {
 
   // Added 08/2026.
   prefs->ClearPref(kWaitingForMultiProfileForcedMigrationTimestamp);
+  brave::MigrateObsoleteLocalStatePrefs(prefs);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -1014,6 +1017,7 @@ void MigrateObsoleteProfilePrefs(PrefService* prefs) {
   // Deprecated 09/2026.
   prefs->ClearPref(kSigninHasAcceptedManagementDialog);
   prefs->ClearPref(kFingerprintingProtectionEnabled);
+  brave::MigrateObsoleteProfilePrefs(prefs);
 }
 
 void MigrateObsoleteUserDefault() {
