diff --git a/net/BUILD.gn b/net/BUILD.gn
index 387b93bfcef8c7a6523454526900346e43aa1b1e..d3ddce71c35d294ec4383a3c809c31283e560573 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1236,6 +1236,7 @@ component("net") {
     "url_request/websocket_handshake_userdata_key.cc",
     "url_request/websocket_handshake_userdata_key.h",
   ]
+  sources += brave_net_sources
 
   if (is_posix || is_fuchsia) {
     sources += [ "base/net_errors_posix.cc" ]
@@ -1263,6 +1264,7 @@ component("net") {
     "//third_party/rust/sfv/v0_15/wrapper",
     "//third_party/simdutf",
   ]
+  deps += brave_net_deps
 
   public_deps = [
     ":chrome_root_store_types",
