diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index 2292d62615f6cc14bbb5097c7fda266de894c1cc..8eebfc291eabb8a20557a9a7261c5c2773638b01 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -319,6 +319,7 @@
     {
       // Interest Group JS API/runtimeflag.
       name: "AdInterestGroupAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       origin_trial_feature_name: "AdInterestGroupAPI",
       implied_by: ["Fledge", "Parakeet"],
       public: true,
@@ -449,6 +450,7 @@
     },
     {
       name: "AIPromptAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       public: true,
       status: {
         "Win": "stable",
@@ -476,6 +478,7 @@
     },
     {
       name: "AIPromptAPIMultimodalInput",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: {
         "Win": "stable",
         "Mac": "stable",
@@ -530,7 +533,7 @@
       origin_trial_feature_name: "AIProofreaderAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -545,7 +548,7 @@
       origin_trial_feature_name: "AIRewriterAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -554,6 +557,7 @@
     },
     {
       name: "AISummarizationAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: {
         "Win": "stable",
         "Mac": "stable",
@@ -588,7 +592,7 @@
       origin_trial_feature_name: "AIWriterAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -1401,7 +1405,7 @@
       // objects that live on the compositor thread.
       // [1] https://drafts.csswg.org/css-animations-2/#timeline-triggers
       name: "CompositorTimelineTrigger",
-      status: "stable",
+      status: "experimental", // Brave: revert when crbug.com/541924666 is fixed.
     },
     {
       name: "CompressionDictionaryTransport",
@@ -1473,6 +1477,7 @@
       // https://github.com/WICG/controlled-frame/blob/main/README.md for more
       // info.
       name: "ControlledFrame",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       public: true,
       status: "stable",
     },
@@ -2983,7 +2988,8 @@
     },
     {
       name: "FencedFrames",
-      base_feature: "none",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
+      base_feature: "FencedFramesRuntime",
       // This helps enable and expose the <fencedframe> element, but note that
       // blink::features::kFencedFrames must be enabled as well, as we require
       // the support of the browser process to fully enable the feature.
@@ -3072,6 +3078,7 @@
     {
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
+      base_feature: "none",
       status: "experimental",
     },
     {
@@ -3086,6 +3093,7 @@
     {
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
+      base_feature: "none",
       status: {"default": "stable"},
     },
     {
@@ -3216,6 +3224,7 @@
     },
     {
       name: "Fledge",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       public: true,
     },
     {
@@ -3983,7 +3992,7 @@
         "Android": "test",
         "default": "",
       },
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -4359,6 +4368,7 @@
     // "experimental" is the support on other platforms.
     {
       name: "MiddleClickAutoscroll",
+      base_feature: "none",
       status: "test",
     },
     // Killswitch for crbug.com/40062462 fix.
@@ -4863,6 +4873,7 @@
     {
       // PARAKEET ad serving runtime flag/JS API.
       name: "Parakeet",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       origin_trial_feature_name: "Parakeet",
     },
     {
@@ -5078,6 +5089,7 @@
       //
       // It also has some feature params defined throughout the codebase.
       name: "Prerender2",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
     },
     {
@@ -5282,6 +5294,7 @@
     // but we still keep this flag for future phase tests.
     {
       name: "ReduceUserAgentMinorVersion",
+      base_feature_status: "enabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
     },
     {
@@ -6731,7 +6744,7 @@
         "ChromeOS": "stable",
         "default": "",
       },
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -6903,6 +6916,7 @@
     },
     {
       name: "UserMediaElement",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       public: true,
     },
@@ -7752,5 +7766,10 @@
       // crbug.com/435623334.
       name: "XSLTSpecialTrial",
     },
+    {
+      name: "BraveIsInTorContext",
+      public: true,
+      base_feature: "none",
+    },
   ],
 }
