diff --git a/third_party/blink/renderer/build/scripts/make_instrumenting_probes.py b/third_party/blink/renderer/build/scripts/make_instrumenting_probes.py
index 6d1ea523aa7fef4c8a7b2887a0b3d23e84bb08ed..7ebde13a9c82cac5161535fc61e0a5c080801110 100644
--- a/third_party/blink/renderer/build/scripts/make_instrumenting_probes.py
+++ b/third_party/blink/renderer/build/scripts/make_instrumenting_probes.py
@@ -319,5 +319,6 @@ def main():
         h_file.close()
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == "__main__":
     main()
