diff --git a/ui/views/widget/widget.h b/ui/views/widget/widget.h
index 0ca8a1679f8c95e68ffb35f63634ba17603c74e2..ccdc54c5a5262659ef33116c34f66d88afe9af1e 100644
--- a/ui/views/widget/widget.h
+++ b/ui/views/widget/widget.h
@@ -663,7 +663,7 @@ class VIEWS_EXPORT Widget : public internal::NativeWidgetDelegate,
 
   // Initializes the widget, and in turn, the native widget. |params| should be
   // moved to Init() by the caller.
-  void Init(InitParams params);
+  virtual void Init(InitParams params);
 
   // Returns the gfx::NativeView associated with this Widget.
   gfx::NativeView GetNativeView() const;
