diff --git a/ui/views/bubble/bubble_dialog_delegate_view.h b/ui/views/bubble/bubble_dialog_delegate_view.h
index da760be83c8a26d7bd8f39cfcf00c0191a56c98f..2ae2c8ec51212cce892ac1a664b908a94ef45c68 100644
--- a/ui/views/bubble/bubble_dialog_delegate_view.h
+++ b/ui/views/bubble/bubble_dialog_delegate_view.h
@@ -877,6 +877,15 @@ class VIEWS_EXPORT BubbleDialogDelegateView : public View,
   gfx::Size GetMaximumSize() const final;
 
  private:
+  friend class ::BraveNewsBubbleView;
+  friend class ::BraveHelpBubbleDelegateView;
+  friend class ::WaybackMachineBubbleView;
+  friend class ::playlist::PlaylistBubbleView;
+  friend class ::SplitViewMenuBubble;
+  friend class ::SidebarItemAddedFeedbackBubble;
+  friend class ::SidebarEditItemBubbleDelegateView;
+  friend class ::SidebarAddItemBubbleDelegateView;
+  friend class ::views::BraveBubbleDialogDelegateView;
   FRIEND_TEST_ALL_PREFIXES(BubbleDelegateTest, CreateDelegate);
   FRIEND_TEST_ALL_PREFIXES(BubbleDelegateTest, NonClientHitTest);
 
