diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index c64c5730430e3df1e65808f081c7ccd3fd9874fb..cfb88408a0f54237c5b710d08c37a5ed91c3f570 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3498,6 +3498,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
